06_modules_lab/01.py

```diff
@@ -1,4 +1,19 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+#Commited by Shlomit

```

Comment:
Please leave the top comments in the file (the ones inside """). They make it easier for everyone opening the file later to understand what's inside